src/main.styl

```diff
@@ -8,7 +8,7 @@ $venetian-red = #D0021B
 $white = #FFFFFF
 
 body
-  font-size 100%
+  font-size 87.5%
```

Comment:
The commit says this should be 14px?  I am confused.

Reply:
The default body font-size for most browsers is 16px so that will generally make our base font-size 14px. To allow users to customize font size (for example if people have low vision and want it really big) we decided to set the base font to be a percent instead of px and then the rest is `rem`

Reply:
I was confused because the commit message said " Set font to be 14px with default browser settings" when the code showed `font-size: 87.5%`  The commit message seemed contradictory.